target/linux/mediatek/dts/mt7981b-confiabits-mt7981.dts

```diff
@@ -124,7 +142,7 @@
 	#address-cells = <1>;
 	#size-cells = <0>;
 
-	switch: switch@0 {
+	switch: switch@31 {
```

Comment:
Sorry for nagging again. Addresses are supposed to be in hex.
```suggestion
	switch: switch@1f {
```

Reply:
Done, thanks